pkg/sdn/plugin/bin/openshift-sdn-ovs

```diff
@@ -9,11 +9,14 @@ tenant_id=$3
 ingress_bw=$4
```

Comment:
Do we need this commit in this PR for the core idling work to land?  If not I'd like it split out given the perf concern raised below.